packages/replay/src/replay.ts

```diff
@@ -625,7 +625,7 @@ export class ReplayContainer implements ReplayContainerInterface {
     // Send replay when the page/tab becomes hidden. There is no reason to send
     // replay if it becomes visible, since no actions we care about were done
     // while it was hidden
-    this._conditionalFlush();
+    this._conditionalFlush({ finishImmediate: true });
```

Comment:
option could be `sync: true`, slightly shorter?

Reply:
Also because the `immediate` naming clashes a bit with `finishImmediate`, which is not actually sync, so IMHO using different naming here would make sense!